src/packages/mikroorm/src/provider/provider.ts

```diff
@@ -165,6 +171,24 @@ export class MikroBackendProvider<D> implements BackendProvider<D> {
 		this.connection = connection;
 		this.addRequestContext();
 		this.connectToDatabase();
+		
+	}
+	private getDbType(): DatabaseType {
+		const driver = this.em.getDriver().constructor.name;
+		switch (driver) {
+			case SqliteDriver.name:
+				return "sqlite";
+			case MySqlDriver.name:
+				return "mysql";
+			case PostgreSqlDriver.name:
+				return "postgresql";
+			default:
+				throw new Error(`This driver (${driver}) is not supported!`);
+		}
+	}
+
+	private gqlToMikro(filter: any) {
+		return gqlToMikro(filter, this.getDbType());
```

Comment:
I'd prefer to get rid of this and just have wherever we call `gqlToMikro` look like this. This is just to avoid the confusion between `gqlToMikro` and `this.gqlToMikro`.